### PR TITLE
Feature/lef 260 list root container contents

### PR DESCRIPTION
**Pull request by roboli**
Necessary changes in Frontify's API module in order for the following to work:

- [List Root Container Contents](https://linear.app/lefthook/issue/LEF-260/list-root-container-contents)
- [List Container Contents](https://linear.app/lefthook/issue/LEF-261/list-container-contents)

Basically:

- Enhance the GraphQL queries legibility
- Implement `listLibraryFolders` for fetching subfolders from a Library
- Implement `listProjectFolders` for fetching subfolders from a Project

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @friggframework/api-module-frontify@0.0.3-canary.176.e134a04.0
  # or 
  yarn add @friggframework/api-module-frontify@0.0.3-canary.176.e134a04.0
  ```
</details>

### CODE REVIEW

File: api-module-library/frontify/api.js [hunk 1]
```diff
@@ -52,48 +52,148 @@ class Api extends OAuth2Requester {
...
+                              __typename
+                            }
+                          }
+                        }
+                      }
+                    }`;
+
+        const response = await this._post(this.buildRequestOptions(ql));
+        return {
+            folders: response.data.library.browse.folders.items
```
Comment (leofmds) [inline api-module-library/frontify/api.js:194]: @roboli What happens if there are no results, the object still goes down to items?

Comment (roboli) [reply api-module-library/frontify/api.js:194]: Returns an empty array coming from Frontify GraphQL api:

```
{
  "data": {
    "library": {
      "browse": {
        "folders": {
          "items": []
        }
      }
    }
  },
  "extensions": {
    "complexityScore": 0
  }
}
```

Comment (roboli) [reply api-module-library/frontify/api.js:194]: Only exception is when trying to get folders from non existent Library or Project (using a bad ID for example). In that case method should throw an error :thinking: 

This from GraphQL api:

```
{
  "errors": [
    {
      "message": "Internal server error",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "library"
      ],
      "extensions": {
        "category": "internal"
      }
    }
  ],
  "data": {
    "library": null
  },
  "extensions": {
    "complexityScore": 0
  }
}
```

Comment (roboli) [reply api-module-library/frontify/api.js:194]: Hmmmm nope, request returns `200` with the error above as the result... I'll work on making methods throw an error/exception if that's the case @leofmds

**Comment by roboli** [comment]
This now handle errors from Frontify @leofmds

**Review by seanspeaks — APPROVED** [review]

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-frontify@0.0.3`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-frontify%400.0.3) :rocket:

**Pull request merged**